src/Todoist.Net/Models/DueDate.cs

```diff
@@ -143,8 +108,98 @@ internal string StringDate
                     return;
                 }
 
-                Date = DateTime.Parse(value, CultureInfo.InvariantCulture);
+                Date = DateTime.Parse(value, CultureInfo.InvariantCulture, DateTimeStyles.RoundtripKind);
             }
         }
+
+
+        /// <summary>
+        /// Creates a new instance of <see cref="DueDate"/> from text (every day; each Monday).
+        /// </summary>
+        /// <remarks>
+        /// See <see href="https://developer.todoist.com/sync/v9/#due-dates">Todoist documentation</see> for more information.
+        /// </remarks>
+        /// <param name="text">The human-readable representation of due date (every day; each Monday)</param>
+        /// <param name="language">The language of the text.</param>
+        /// <returns>The created <see cref="DueDate"/> instance.</returns>
+        public static DueDate FromText(string text, Language language = null) => new DueDate
+        {
+            Text = text,
+            Language = language
+        };
+
+        /// <summary>
+        /// Creates a full-day <see cref="DueDate"/> in the format of <c>YYYY-MM-DD</c> (<see href="https://datatracker.ietf.org/doc/html/rfc3339">RFC 3339</see>).
```

Comment:
I think this part

> in the format of <c>YYYY-MM-DD</c> (<see href="https://datatracker.ietf.org/doc/html/rfc3339">RFC 3339</see>).

Is not useful for the library users. We should let user know what's the difference between each date type.